src/test/appdistro/client.spec.ts

```diff
@@ -1,29 +1,27 @@
 import { expect } from "chai";
 import { join } from "path";
+import * as fs from "fs-extra";
 import * as rimraf from "rimraf";
 import * as sinon from "sinon";
 import * as tmp from "tmp";
 
-import {
-  AppDistributionClient,
-  AppView,
-  UploadStatus,
-  UploadStatusResponse,
-} from "../../appdistribution/client";
+import { AppDistributionClient, BatchRemoveTestersResponse } from "../../appdistribution/client";
 import { FirebaseError } from "../../error";
 import * as api from "../../api";
 import * as nock from "nock";
-import { Distribution, DistributionFileType } from "../../appdistribution/distribution";
+import { Distribution } from "../../appdistribution/distribution";
+import { describe } from "mocha";
```

Comment:
This line shouldn't be necessary. Please remove.

Reply:
Done.